sandbox/sandboxapp.cpp

```diff
@@ -12,6 +13,13 @@ SandboxApp::SandboxApp(int &argc, char **argv)
       m_handler(new Handler(this))
 {
     connect(m_handler, &Handler::restartQml, this, &SandboxApp::restartEngine, Qt::QueuedConnection);
```

Comment:
@borismelnik what's the purpose of `m_handler`, I see the usage in the line 46, but except in this connection I don't see anywhere that the registered context property "app" is used nor `restartQml` signal sending?

Reply:
yeah its old code